product_docs/docs/pem/9/managing_certificates_copy.mdx

```diff
@@ -0,0 +1,283 @@
+---
+title: "Configuring Secure Connection with PEM Server"
+navTitle: "Certificates Copy"
+legacyRedirectsGenerated:
+  # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
+  - "/edb-docs/d/edb-postgres-enterprise-manager/user-guides/administrators-guide/8.0/managing_certificates.html"
+# This file is moved from pem_admin/04_managing_certificates
+redirects:
+- /pem/latest/pem_admin/04_managing_certificates/
+---
+PEM, by default, supports secured SSL/TLS connections and acts as its own certificate authority (CA) to generate certificates and keys for the PEM server and agent. 
+
+The SSL CA certificates and keys for the PEM server and agent gets generated during the PEM installation. These certificates and keys encrypt the connections between them. You can replace the PEM self-signed CA certificate and key with trusted third-party CA certificates.
+
+## PEM self-signed CA certificate framework
+
+During the PEM Server configuration, the script generates self-signed CA certificates and key files for the PEM backend database server. Then, it configures the backend database server to use these certificates and keys for authentication and encryption purposes for the agent connections.
```

Comment:
The PEM server configuration script generates self-signed CA certificates and key files for the PEM backend database server. The backend database server uses these certificates and keys to authenticate and encrypt the agent connections.